internal/abci/cometbft/peers.go

```diff
@@ -1,10 +1,183 @@
 package cometbft
 
 import (
+	"context"
+	"fmt"
+	"sync"
+
 	"github.com/cometbft/cometbft/crypto/ed25519"
 	"github.com/cometbft/cometbft/p2p"
+	"github.com/kwilteam/kwil-db/common/sql"
+	"github.com/kwilteam/kwil-db/internal/sql/versioning"
+)
+
+const (
+	p2pSchemaName = "peers" // TODO: Umm, should this be part of the chain metadata schema?
+
+	p2pStoreVersion = 0
+
+	initPeersTable = `CREATE TABLE IF NOT EXISTS ` + p2pSchemaName + `.peers (
+		peer_id TEXT PRIMARY KEY
+	);`
+
+	addPeer = `INSERT INTO ` + p2pSchemaName + `.peers ` + `VALUES ($1) ` + `ON CONFLICT (peer_id) DO NOTHING;`
```

Comment:
I think if the peer already exists, we should return an error. If I was a user, I would expect some sort of error like `"%s" is already whitelisted as a peer`